base/shell/explorer/lang/nl-BE.rc

```diff
@@ -0,0 +1,200 @@
+    CONTROL          "Start Menu instellingen", IDC_CLASSICSTART_SETTINGS, "SysTreeView32", WS_BORDER | 0x00000001, 8, 139, 218, 72, WS_EX_CLIENTEDGE
+    DEFPUSHBUTTON    "OK", IDOK, 122, 220, 50, 14
+    PUSHBUTTON       "Annuleren", IDCANCEL, 176, 220, 50, 14
+END
+
+IDD_FILENAME_WARNING DIALOGEX 0, 0, 272, 81, 0
+STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | DS_CENTER | WS_POPUPWINDOW | WS_DLGFRAME
+CAPTION "Bestandsnaam waarschuwing"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    DEFPUSHBUTTON    "&Negeren", IDIGNORE, 215, 60, 50, 14
+    PUSHBUTTON       "&Aanpassen", IDC_FILE_RENAME, 159, 60, 50, 14
+    LTEXT            "", IDC_STATIC, 7, 51, 258, 1, NOT WS_GROUP
+    AUTOCHECKBOX     "&Deze check niet uitvoeren bij het opstarten.", IDC_NOTCHECK, 7, 62, 139, 10
+    ICON             32515, IDC_STATIC, 7, 7, 21, 20
+    LTEXT            "Er bestaat een bestand of folder op uw computer genaamd %s, dat ervoor zou kunnen zorgen dat bepaalde applicaties nit correct werken. De naam veranderen naar %s zou dit probleem oplossen. Wilt u de naam nu aanpassen?", IDC_STATIC, 34, 7, 224, 37
+END
+
+IDD_MODERNSTART_ADVANCED DIALOGEX 0, 0, 251, 198, 0
+STYLE DS_SETFONT | WS_CAPTION | WS_CHILD | WS_DISABLED
+CAPTION "Geavanceerd"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    GROUPBOX         "Start menu instellingen", IDC_STATIC, 7, 7, 238, 42
+    AUTOCHECKBOX     "&Submenus openen wanner ik mijn muis erboven houd", IDC_AUTOOPEN, 14, 21, 224, 10
+    AUTOCHECKBOX     "Nieuw geïnstalleerde programmas markeren.", IDC_HIGHLIGHT, 14, 35, 224, 10
+    LTEXT            "Start &menu onderdelen:", IDC_STATIC, 7, 53, 238, 8
+    CONTROL          "StartMenuInstellingen", IDC_ITEMS, "SysTreeView32", WS_BORDER | 0x00000001, 7, 63, 238, 67, WS_EX_CLIENTEDGE
+    GROUPBOX         "Recente Bestanden", IDC_STATIC, 7, 133, 238, 56
+    LTEXT            "Selecteer deze optie om vlugge toegang te bieden tot bestanden die u recentelijk hebt geopend. Deze lijst wissen verwijdert de bestanden niet.", IDC_STATIC, 14, 144, 224, 21
+    AUTOCHECKBOX     "Toon mijn meest recentelijk geopende bestanden", IDC_RECENTLY, 14, 168, 147, 10
+    PUSHBUTTON       "&Lijst wissen", IDC_CLEAR, 182, 168, 55, 14
+END
+
+IDD_MODERNSTART_GENERAL DIALOGEX 0, 0, 251, 198
+STYLE DS_SETFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_DLGFRAME
+CAPTION "Algemeen"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    GROUPBOX         "Selecteer een grootte voor programma iconen.", IDC_STATIC, 7, 7, 238, 37
+    ICON             IDC_LARGEICON, 17, 18, 20, 20, SS_CENTERIMAGE
+    ICON             IDC_SMALLICON, 151, 18, 20, 20, SS_CENTERIMAGE
+    AUTORADIOBUTTON  "&Grote iconen", IDC_CHOOSELARGE, 43, 25, 53, 10
+    AUTORADIOBUTTON  "&Kleine iconen", IDC_CHOOSESMALL, 177, 25, 51, 10, NOT WS_TABSTOP
+    GROUPBOX         "Programmas ", IDC_STATIC, 7, 49, 238, 70
+    LTEXT            "Het start menu bevat shortcus naar de programmas die u het meeste gebruikt. deze lijst verwijderen wist deze programmas niet.", IDC_STATIC, 14, 63, 224, 20
```

Comment:
instead of `shortcus` (which is spelled wrong) we tend to use `snelkoppelingen`
`deze` is the start of a new sentence, so should be capitalized